tests/sample.js

```diff
@@ -1,6 +1,7 @@
 import * as dotenv from "dotenv";
 
-import { places } from "../build/index.js";
+import { ngd } from "../build/ngd.js";
+import * as filters from "../build/filters/filters.js";
 
 dotenv.config();
```

Comment:
I'm not sure i like this file being in the package. it's just a playground for us

Reply:
untracked and added to gitignore